src/services/timesheet.js

```diff
@@ -0,0 +1,43 @@
+const { Timesheet } = require("../models");
+const { v4: uuidv4 } = require("uuid");
+const { Op } = require("sequelize");
+
+async function findAll() {
+  return await Timesheet.findAll();
+}
+
+async function createNew(timesheetObject) {
+  const newTimesheet = { ...timesheetObject, id: uuidv4() };
+  return await Timesheet.create(newTimesheet);
+}
+
+async function deleteExisting(id) {
+  return await Timesheet.destroy({ where: { id } });
+}
+
+async function findOne(id) {
+  return await Timesheet.findOne({ where: { id } });
+}
+
+async function findAllWhere(dateString) {
+  return await Timesheet.findAll({
```

Comment:
No need to await the promise here, since we're returning the promise and awaiting it in the calling function either way
```suggestion
  return Timesheet.findAll({
```

Reply:
I will fix it

Reply:
Everything fixed I think.